notebooks/2018-11-27-Value-when-other-is-max.ipynb

```diff
@@ -0,0 +1,549 @@
FyzyllhxA1VY6m1iqGre8kUdFxKEBG81tpJE7lWqjTpODC1rrTo/vmffMUmsMw0YHpPUS5lepo7ZTu+hkAILeAqHru5lGYeSwVb/QExktzWF2G4uFsbB6dXvTWL1cTmPuxzxp7xgV/NvbQZppn/Rs/DgCY2U/xk/2//Jqx695wJRvd4kriiJC3UcORySGA6AhiFzWe4H4QoiJ537Z7i2vsCMoxijJ37UeUe0FilQGLWkBonhW6+fDjFqWy4BovUtUhNOexXTBcyhhUwUE/R4jOh5xdWMS+KrLjxuZfytZhNFLsHqFGmPLLpmeMnema53dA97eBh+4wdv3pFxt78K83Grtvg/es4m5qzXPs6jGBm1IfSwiwdTtLDsDqLmoj1ZGefk8toHs13SMO5x3HUMmwwlU3ak0eICL+SXGAuJRWHVE6qp8da7tAIBAIBAKBYPwhUf8CgUAgEAgEAsEkwkH0fVwTTRAVvR4VxXrc+caeNo+it0uMOimOeNQDj+7nbgC8PJUhIfkt9zzh/b+CIiZHeinyevqJc4w95zESiR7a40WCLr/qh6bspD+/ms2a7VNADbnEziMir53UoStyPiLHeKxc59aGCaL3XdSSRSTbRZ1GztOlAMD/sMyPi//nO4+w9pG64K0AgJHbvl9WBgDFW66haVzwRq9fTgdnKMI4tB9Fytmtaxq97Yw2KzW0Gzt99muNXc/oySCCusRyk9cdtpC2P/wnGuPES9heBfNglGycY2jb7qJlo5IGRFCEcUT+rW4AlSpV8H55caXUqKUP5zWW4NpL4moRZ4xE+xSRYMGJCudmEEclJuK+UKprNXa6l+7x/Pq0wqX2kSd3heGnPDWPTCc9I7hb2a4bSDGm6WhSImj0n1s9q563Dp1rqi+z1935d1N27Lk0XvMcul+s/OXjVJ717s9nXfsNU0Z3njHOX8Szw4mo+hEqOAdiRL/WOGDkpKoN+aIqEAgEAoFAIJiUiPWiqpRapJS6Qym10v/7OD9TlUAgEAgEAoFgnCHyVGPjewA+CuC7AKC1flwp9XMAXxiviSWGRjnN6xBtjorGLdaz/MojROXOPI0okHTO62NgB+VMztRTFGTT4XON/dzv7jT27jVePvX2hUQRDe2hKM9Z559o7I5fPWbsmuYI6iiKEuCJAhz55q3i2hwRdFgs8XQ+nq3fKDj2wxnRG8zNFSldId2fKFd46HjbXReC+afPewNrRmNkLvrHsjYlJvKv2BiZHSTszY9LYephXr98/bPjqRndv+/7/27sprdf5fXFBcrPfB31USB1CjVCbiymvyrTbJUKyXME6zCpW4qJXq/C3CKF8mNExVtdFJIeH8s8nPthUXKoBiKp4TiuFpUqikT1xWFxJeFzL2boGaBW3Wfs1NFnU3mQjIC5A+g0uYfxeWRmzKfirFe/2M2UQ/ro+dN0xCJW3mPs4T2eokCJJTSYcjglBBjaQ4oDA9u8qP2mGY2mbNHrLjD28zfdY+y6KTT/kz54EQCg0LHAuh9R58+pAMH/iKPEMMYYbjWbA4dYPkSZ/9jUf73W+sFRZRWmJhIIBAKBQCAQCKIR94tqt1LqcPgfwZRSrwawddxmJRAIBAKBQCAA4GemOkQ/qcZ9UX0fgGsALFZKbQawDsAbx21WlUApk0PZfOIv2QWXlSW6k1Mv6X3dxu5aShLHLYwuGe7xRJB5xGRxiGjWLXfSB+ju1buNXRj0xt74ANE3M0+iMVSOqKN0jmjZWx71RJvPaKfxwCK6Q/QFp8eDzTz6m0egRjEoPPKeRbRH5mZ2RXfzdv55cNKM/KIM5uGK+nfA6uYRpRAA2CkpJ31VYSIEvi8Rkc6clo+ivbZ8/3+MvfJny43dMtdTpVj6mXdQ5VOWWefA1zKn/E0Zo/v5/ofmafqLcc4iROcrRsT5q4YbQZIoexdtb20eQ4WgYkQJ/jsp9eBcVzmCOgn1H+eajEoQkmQOvI9ieTICjswwJXJMH04JORRTACjVe25fPBEIVwYp1VKymBRfL/69TPP78MgQDc7u+6UhSk4TCP2ns3QvrJnK3Nvy9GzoWOK5D/Q+T8+s7Q88Yew7frfa2Ccu6TR20zJPocR1hEP34UrXS+gZZ3FB4TYfr5IEQKPrCyYUsV5UtdbPAbhQKdUAIKW17otqIxAIBAKBQCCoDg7N76kRL6pKqX9xlAMAtNbfsG0XCAQCgUAgEAj2F1FfVAMO4ggAJwMIEghfCuDu8ZrUfiP4ZO+I9NY2NwAeQb6W6NKtK8gVl1Mngbh/MU/UTO96qstzJQ/3Ek3aOM2jeBa/eokpW/Xrh4394JeuM3augdwRjvGj/tuYWkB6LbkXFBechtHg+/nUG19j7KN//HOqw6NNTaG22slEuxkd7lIZsNDySQTaQ0oOEfnPnZSraz4WOtQZpR1KN54um5uzskX8XVvygANhqj3Yl9x6WqelqfOMfd81JNa9uo8o/Jn+OjxpKeUEdx2LnY89Z+y2IY9A0TUN1rrOmExf4cAZWZ9AHN8JyzmJlafdVtcFvnYi1pkVSffftk9Ruc0RrWbibmg5Bk5KfYK+6SRNYrC/7giu9ml2jAsWpRF2ngbbKQJ+27+9xdjzP/KJMYcOXAMAoMRcAgL3rXQrKWsUNq1lDcklIN9L1L9Ke8eo7WhSEBjcvtPYfRsosUzNFC/af4g9s4pDZM+so/1vW0juA8Gcna5SrnVfofqGFa5rM8L9y+kyMAmTAhyqKVTHvLNprT8LAEqpWwGcGFD+SqmrAPxm3GcnEAgEAoFAIBB5qgjMAcB/Po4AmFf12QgEAoFAIBAIBD7ickU/BfCgUup6eCzlKwH8ZNxmVRG0PbovQESUOhdGv/kVlHSraTpRL3uf78FodK8mCiVbS4dzoJsipaf40dYAsOjyUwEAe9ZuNmXFEZpP7RSaZ/92ong2DHi0z6whOwUYEo/2afcUo4u3+aoBAHBUnCh7U1YhJcNoKOWg1210JndFCEWWB2W8rmvsBN
```

Comment:
Did you intend this to be kept in? You didn't use it for the other implementations.
If you do want it in, maybe explain what it does: "We will use `%%time` to tell us the CPU time taken."